app/assets/javascripts/hqmf_util.js.coffee

```diff
@@ -998,12 +998,12 @@ dateSortAscending = (a, b) ->
   a.timeStamp().getTime() - b.timeStamp().getTime()
 @dateSortAscending = dateSortAscending
 
-applySpecificOccurrenceSubset = (operator, result, range, calculateSpecifics) ->
```

Comment:
I looked at some old diffs, it looks like the original version checked in didn't use the variable, so it must have been some idea that never went anywhere.